submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/db/lock/DistributedLockExecutor.kt

```diff
@@ -56,7 +56,7 @@ internal class DistributedLockExecutor(
     )
 
     private companion object {
-        val DEFAULT_EXPIRATION = Duration.ofMinutes(5)
+        val DEFAULT_EXPIRATION = Duration.ofHours(3)
```

Comment:
extended lock time to avoid two submission being processed concurrently after lock expire (we need refresh lock feature)